easybuild/easyconfigs/b/Bowtie/Bowtie-1.2.2-foss-2018b.eb

```diff
@@ -0,0 +1,22 @@
+name = 'Bowtie'
+version = '1.2.2'
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+ It aligns short DNA sequences (reads) to the human genome."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True, 'cstd': 'gnu++98'}
+
+sources = ['%(namelower)s-%(version)s-src.zip']
+source_urls = ['http://download.sourceforge.net/bowtie-bio/']
+checksums = [
+    '806d618d073107e75dec6c3a61fc9ce4a1eb91657e75cb1bfa9ca2bf926482ca',  # bowtie-1.2.2-src.zip
```

Comment:
@smoors Please make this a single line (since there's only one source & no patches)